src/core/server/metrics/event_loop_delays/event_loop_delays_monitor.ts

```diff
@@ -6,10 +6,23 @@
  * Side Public License, v 1.
  */
 
+import moment from 'moment';
 import type { IntervalHistogram as PerfIntervalHistogram } from 'perf_hooks';
 import { monitorEventLoopDelay } from 'perf_hooks';
 import type { IntervalHistogram } from '../types';
 
+/**
+ * Nanosecond to milisecond conversion unit
+ */
+export const ONE_MILLISECOND_AS_NANOSECONDS = 1000000;
```

Comment:
```suggestion
export const ONE_MILLISECOND_AS_NANOSECONDS = 1_000_000;
```

Reply:
Definitely easier to parse by eye. Nice suggestion!